jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/core/timers/DslConstantThroughputTimerTest.java

```diff
@@ -0,0 +1,94 @@
+package us.abstracta.jmeter.javadsl.core.timers;
+
+import org.junit.jupiter.api.*;
+import us.abstracta.jmeter.javadsl.*;
+import us.abstracta.jmeter.javadsl.codegeneration.*;
+import us.abstracta.jmeter.javadsl.core.*;
+import java.time.*;
+import static org.assertj.core.api.Assertions.*;
+import static us.abstracta.jmeter.javadsl.JmeterDsl.*;
+
+public class DslConstantThroughputTimerTest extends JmeterDslTest {
+
+  @Test
+  public void checkCorrectSamplesConstantThroughputTimerCalMode1() throws Exception {
```

Comment:
Prefer explicit names instead of CalMode1, 2, etc.

Reply:
fair enough, wanted to keep the method names a sensible length